2-extra/1-quote-generator-revisited/quotes.js

```diff
@@ -0,0 +1,530 @@
+      "Our lives begin to end the day we become silent about things that matter.",
+    author: "Martin Luther King Jr.",
+  },
+  {
+    quote: "Do what you can, where you are, with what you have.",
+    author: "Teddy Roosevelt",
+  },
+  {
+    quote:
+      "If you do what you’ve always done, you’ll get what you’ve always gotten.",
+    author: "Tony Robbins",
+  },
+  {
+    quote: "Dreaming, after all, is a form of planning.",
+    author: "Gloria Steinem",
+  },
+  {
+    quote:
+      "It’s your place in the world; it’s your life. Go on and do all you can with it, and make it the life you want to live.",
+    author: "Mae Jemison",
+  },
+  {
+    quote:
+      "You may be disappointed if you fail, but you are doomed if you don’t try.",
+    author: "Beverly Sills",
+  },
+  {
+    quote: "Remember no one can make you feel inferior without your consent.",
+    author: "Eleanor Roosevelt",
+  },
+  {
+    quote: "Life is what we make it, always has been, always will be.",
+    author: "Grandma Moses",
+  },
+  {
+    quote:
+      "The question isn’t who is going to let me; it’s who is going to stop me.",
+    author: "Ayn Rand",
+  },
+  {
+    quote:
+      "When everything seems to be going against you, remember that the airplane takes off against the wind, not with it.",
+    author: "Henry Ford",
+  },
+  {
+    quote:
+      "It’s not the years in your life that count. It’s the life in your years.",
+    author: "Abraham Lincoln",
+  },
+  {
+    quote: "Change your thoughts and you change your world.",
+    author: "Norman Vincent Peale",
+  },
+  {
+    quote:
+      "Either write something worth reading or do something worth writing.",
+    author: "Benjamin Franklin",
+  },
+  {
+    quote: "Nothing is impossible, the word itself says, “I’m possible!”",
+    author: "–Audrey Hepburn",
+  },
+  {
+    quote: "The only way to do great work is to love what you do.",
+    author: "Steve Jobs",
+  },
+  {
+    quote: "If you can dream it, you can achieve it.",
+    author: "Zig Ziglar",
+  },
+];
+
+let button = document.querySelector("button");
+let quote = document.querySelector(".quote");
+let author = document.querySelector(".author");
+
+function ahowQuote() {
+  let arrquotes = pickFromArray(quotes);
+  quote.innerHTML = arrquotes.quote;
+  author.innerHTML = arrquotes.author;
+}
+
+button.addEventListener("click", function () {
+  ahowQuote();
+});
+// 💫💫💫💫💫💫💫💫💫💫💫💫💫💫
+
+const whiteDiv = document.querySelector(".whiteDiv");
+const customSwitch1 = document.querySelector("#customSwitch1");
+// let customSwitch2 = document.querySelector("#customSwitch2");
+let show;
+let p;
+customSwitch1.addEventListener("click", function () {
+  if (customSwitch1.checked) {
+    quote.textContent = "";
+    p = document.createElement("p");
+    p.textContent = "auto-play:ON";
+    p.style.color = "green";
+    quote.classList.add("quote-dark");
+    whiteDiv.appendChild(p);
+    show = setInterval(() => {
+      ahowQuote();
+    }, 1000);
```

Comment:
Did you reduce this to 1 second for testing purposes? You should set it to 60 seconds.